corehq/apps/reports/tasks.py

```diff
@@ -47,11 +48,12 @@ def update_calculated_properties():
         get_domains_to_update_es_filter()
     ).fields(["name", "_id"]).run().hits
 
-    update_calculated_properties_in_chunks.chunks(domains_to_update, 5000).apply_async(queue='background_queue')
+    for chunk in chunked(domains_to_update, 5000):
+        update_calculated_properties_for_domains.delay(list(chunk))
```

Comment:
Does this need to be a list rather than stay a generator? If so is that for serialization reasons?

Reply:
Tested locally and it is unnecessary.